lte/gateway/c/session_manager/ShardTracker.h

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <string>
+#include <utility>
+#include <vector>
+#include <map>
+
+namespace magma {
+
+// Shards represent groups of UEs placed into buckets of
+// a certain size, to make polling more manageable
+class ShardTracker {
+  ShardTracker();
+
+  // add UE to shards based on availability
+  int add_ue();
+
+  // remove UE from shard
+  void remove_ue(int shard_id);
+
+ private:
+  std::vector<int> shards;
```

Comment:
Can we add comments on what this contains? What is the int in this case? Similarly we could probably use a better variable name than shards here. 

The PR description says `An approach is to create a shard class that stores a map of shard IDs to a vector of UEs with unique IMSIs`, but I can't tell if that's outdated or not.

Reply:
Thanks for the PR description catch, that should definitely be updated.